code/go/internal/validator/semantic/types.go

```diff
@@ -33,41 +34,50 @@ type field struct {
 	Fields fields `yaml:"fields"`
 }
 
-type validateFunc func(fieldsFile string, f field) ve.ValidationErrors
+type fieldFileMetadata struct {
+	packageType  string
+	packageName  string
```

Comment:
`packageType` and `packageName` are not used at the end, right? Do we need them? And `filePath` is only used once, and maybe we can use `fullFilePath` there.

Reply:
Sure, I'll check that, and I'll remove them if they are not needed now. :+1:

Reply:
Removed `pacakgeType` and `packageName` fields